.github/workflows/static-test.yml

```
@@ -0,0 +1,23 @@
+name: static-test
```

I would prefer

```suggestion
name: static-tests
```

btw.